Tone/signal/Pow.ts

```diff
@@ -0,0 +1,85 @@
+import { Signal, SignalOptions } from "./Signal";
+import { WaveShaper, WaveShaperMappingFn } from "./WaveShaper";
+import { optionsFromArguments } from "Tone";
+
```

Comment:
Add an interface which extends ToneAudioNodeOptions called PowOptions which might look something like this: 

```typescript
interface PowOptions extends ToneAudioNodeOptions {
  value: number;
}
```